### ISSUE TITLE
proposal: make foxdot fully working in command line without TK as dep

### ISSUE THREAD

**Issue by federicobriata** [Qirky/FoxDot#222]
I'm trying to make it work on a remote embedded Gnu/Linux system where I don't have a screen but just a serial port or remote terminal.
This is the error I get:

```
# python3 -m FoxDot
Warning: Could not fetch info from SCLang server. Using defaults...
Traceback (most recent call last):
  File "/usr/lib/python3.5/runpy.py", line 183, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/usr/lib/python3.5/runpy.py", line 142, in _get_module_details
    return _get_module_details(pkg_main_name, error)
  File "/usr/lib/python3.5/runpy.py", line 109, in _get_module_details
    __import__(pkg_name)
  File "/usr/lib/python3.5/site-packages/FoxDot/__init__.py", line 110, in <module>
    from .lib import *
  File "/usr/lib/python3.5/site-packages/FoxDot/lib/__init__.py", line 26, in <module>
    from .Workspace import get_keywords
  File "/usr/lib/python3.5/site-packages/FoxDot/lib/Workspace/__init__.py", line 3, in <module>
    from .Editor import *
  File "/usr/lib/python3.5/site-packages/FoxDot/lib/Workspace/Editor.py", line 18, in <module>
    from .tkimport import *
  File "/usr/lib/python3.5/site-packages/FoxDot/lib/Workspace/tkimport.py", line 4, in <module>
    from tkinter import *
  File "/usr/lib/python3.5/tkinter/__init__.py", line 35, in <module>
    import _tkinter # If this fails your Python may not be configured for Tk
ImportError: No module named '_tkinter'
```

**Comment by kfrncs** [comment]
just dropping in a vote that I'd love for FoxDot to be fully useable via SSH on a headless device.

**Comment by MrMebelMan** [comment]
You can import FoxDot as an ordinary Python module boisss
```python3
from FoxDot import *
k1 >> play('S')
```

**Comment by ryan-kirkb** [comment]
I believe that it will try and import tkinter, even not using the GUI. Which isn't ideal if you don't have a display i.e. connecting via ssh. I have some free time tonight so will look into it.

**Comment by Qirky** [comment]
I believe that it will try and import tkinter, even not using the GUI. Which isn't ideal if you don't have a display i.e. connecting via ssh. I have some free time tonight so will look into it.

**Comment by Qirky** [comment]
Alright, should be working fine when importing or using the `--pipe` flag, which you'll need if you want to run it via the cli.

**Comment by federicobriata** [comment]
to avoid other getting crazy, the fix for this was released with v0.8.4, thanks @Qirky !
https://github.com/Qirky/FoxDot/commit/84c148f1a8daf416f7fa45bbfd1dbd60145bfc4a
